### PR TITLE
[plugin.video.greenpeace] 1.1.8

### PR DESCRIPTION
**Pull request by jamontes**
Hi,

This is a new update of plugin.video.greenpeace under gotham repo.

Changelog:
1.1.8 (2016.05.07)
- Updated Vimeo scraper due to Vimeo website changes.
- Updated language files from Transifex.
- Fixed list of supported languages (credits: razzeee).

Thanks a lot in advance and best regards,

jamontes.

### CODE REVIEW

File: plugin.video.greenpeace/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+*.pyc
```
Comment (phil65) [inline plugin.video.greenpeace/.gitignore]: please remove this file.

Comment (jamontes) [inline plugin.video.greenpeace/.gitignore]: Hi Pil65,

I'll would like to do it properly from my command line, but not sure i'm not breaking the "just one PR per commit" rule or (even worse) the whole local repo. I've deleted the ".gitignore" file from my local repo before by using "git rm .gitignore", but i cannot dare to do an --amend commit and re-push to my fork before reply this PR.

Please, if you can help me on this (i only use git from the command line) to do it in the right way i'll very appreciate it and i'll make it sure of avoid it for next PRs.

Many thanks in advance and best regards,
jamontes.

Comment (MartijnKaijser) [inline plugin.video.greenpeace/.gitignore]: there shouldn't really be any issue with addons having the gitignore file.

Comment (phil65) [inline plugin.video.greenpeace/.gitignore]: ok, then ignore my comment.

Comment (jamontes) [inline plugin.video.greenpeace/.gitignore]: OK, thanks to you both for your help.

Best regards,
jamontes.

**Pull request merged**